demos/people_counter_demo/main.py

```diff
@@ -109,7 +110,11 @@ def postprocess(pred_boxes: np.ndarray, pred_masks: np.ndarray, input_size: Tupl
     # create detections in supervision format
     det = sv.Detections(xyxy=pred[:, :4], mask=masks, confidence=pred[:, 4], class_id=pred[:, 5])
     # filter out other predictions than selected category
-    return det[det.class_id == category_id]
+    det = det[det.class_id == category_id]
+
+    # Convert detections to list of dictionaries
+    detection_list = [{"bbox": d[0].tolist(), "confidence": d[1]} for d in zip(det.xyxy, det.confidence)]
+    return detection_list
```

Comment:
Is this required if you convert it again in unique_identifier? Let's leave this function as it is and make all necessary changes in unique_identifier